dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/PIAntiWindup.mo

```diff
@@ -0,0 +1,75 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```

Comment:
Adapt copyright statement here. It was quite perfect on this point all over the PR ;)

Reply:
Fixed